.eslintrc

```diff
@@ -17,7 +17,14 @@
     "template-curly-spacing": "off",
     "indent": "off",
     "linebreak-style": 0,
-    "no-console": "off"
+    "no-console": "off",
```

Comment:
I noticed that you have made modifications to the ESLint rules. Perhaps we should make these changes in a separate PR and leave this PR to the add-admin-detail feature only.